django/applications/catmaid/control/landmarks.py

```diff
@@ -9,12 +9,11 @@
 from django.shortcuts import get_object_or_404
 from django.utils.decorators import method_decorator
 
-from catmaid.control.authentication import (requires_user_role, can_edit_or_fail,
-        can_edit_all_or_fail)
-from catmaid.control.common import (get_request_list, get_class_to_id_map,
-        get_relation_to_id_map, get_request_bool, get_request_list)
-from catmaid.models import (Class, ClassInstance, ClassInstanceClassInstance,
-        Relation, Point, PointClassInstance, UserRole)
+from catmaid.control.authentication import requires_user_role, can_edit_or_fail, can_edit_all_or_fail
+from catmaid.control.common import get_request_list, get_class_to_id_map, get_relation_to_id_map, get_request_bool
+from catmaid.models import (
+    Class, ClassInstance, ClassInstanceClassInstance, Relation, Point, PointClassInstance, UserRole,
+)
```

Comment:
Since I have seen similar changes in other files, I wonder: does this the new flake8 config change the preferred line width? While I don't have a strong opinion on this (anymore), I always found 80 to be useful/readable in many situations, like in diffs (see above) or with having multiple editor columns to avoid confusing line wrapping. Apart from the line width, I feel it is nice to have consistency in long import statements,. Before all three import lines used the same tuple format for longer imports, but now only the last ones does. What makes them different?

Reply:
Oh, I suppose the rule is now that nothing should follow an opening parenthesis/bracket/brace?

Reply:
In the editorconfig, we use 120, so that's what I used in the flake8 config. I would also prefer it shorter, but I thought I'd stick with what we had for the time being.

That tuple form is necessary here to prevent hitting the 120-column limit. It's possible I removed the tuple form where it wasn't necessary.

Reply:
And I guess I should have read all files to the end first, the `setup.cfg` file config answers my line width question. I am okay with 120 characters as upper limit.

Reply:
> Oh, I suppose the rule is now that nothing should follow an opening parenthesis/bracket/brace?

My strategy is generally "in case of ambiguity, do what `black` does". I think the logic is that if you can't fit everything in parentheses in the same line as the parentheses, then the parentheses go on lines 0 and 2 and the arguments on line 1. If you can't fit all of the arguments on one line, they should each go on their own line. This last can add quite a lot of verticality to the code and I sometimes ignore it if the arguments have some natural grouping. e.g.

```python
my_short_tuple = (a, b, c)
my_long_tuple = (
    things, more_things, a_lot_more_things, so_many_things,
)
my_very_long_tuple = (
    a_really_long_name,
    an_even_longer_name,
    why_are_these_names_so_long_anyway,
)
my_grouped_tuple = (  # not black-compliant
    hat_shape, hat_colour, hat_size,
    shirt_colour, shirt_size,
)
```

It may be that I'm just used to reading `black`ened code now but it makes sense to me to try to fit all members of a tuple on one line, and enclose that line in parens, rather than prioritising putting some members on the same line as one of the parens. Same as putting blocks of code in braces in other languages.

Anyway, these are insignificant minutiae.